collector/net.go

```diff
@@ -0,0 +1,178 @@
+// returns data points from Win32_PerfRawData_Tcpip_NetworkInterface
+
+// https://technet.microsoft.com/en-us/security/aa394340(v=vs.80) (Win32_PerfRawData_Tcpip_NetworkInterface class)
+
+package collector
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"regexp"
+
+	"github.com/StackExchange/wmi"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func init() {
+	Factories["net"] = NewNetworkCollector
+}
+
+var (
+	nicWhitelist = flag.String("collector.network.nic-whitelist", ".+", "Regexp of NIC:s to whitelist. NIC name must both match whitelist and not match blacklist to be included.")
+	nicBlacklist = flag.String("collector.network.nic-blacklist", "", "Regexp of NIC:s to blacklist. NIC name must both match whitelist and not match blacklist to be included.")
```

Comment:
@brian-brazil I missed your comment. Patch updated to remove default blacklist "isatap.+"